scripts/build/Platform/Windows/build_config.json

```diff
@@ -213,19 +213,38 @@
       "ASSET_PROCESSOR_PLATFORMS": "pc,server"
     }
   },
-  "periodic_test_profile_vs2019_pipe": {
+  "awsi_test_profile_vs2019_pipe": {
     "TAGS": [
       "nightly-incremental",
       "nightly-clean"
     ],
     "steps": [
       "awsi_deployment",
-      "periodic_test_profile_vs2019"
```

Comment:
is the periodic_test_profile_vs2019 step being re-added somewhere else?

Reply:
periodic_test_profile_vs2019 doesn't need to be in a pipe any more. I reverted the changes Shirang made in the following PR since the aws_deployment job doesn't need to happen before periodic_test_profile_vs2019 after I remove all the AWSI automation tests to a separate test suite and pipe.

https://github.com/o3de/o3de/pull/3397